src/app/public/modules/code-examples/code-examples-editor.service.ts

```diff
@@ -223,7 +208,8 @@ export class AppComponent {}`;
   bootstrap: [
     AppComponent
   ],
-  providers: [${theme === SkyDocsCodeExampleTheme.Modern ? '\n    SkyThemeService,' : ''}
+  providers: [
+    SkyThemeService,
```

Comment:
I'm worried that this will communicate to the consumer that `SkyThemeService` is required for all component implementations, even "default" ones. Is that the intent?

Reply:
Related: does this mean that the input box component doesn't work in `default` theme? In other words, if an Angular consumer pulled in the box component (without the the Theme service) it wouldn't work?

Reply:
Builder always provides `SkyThemeService`, so I asked Alex to make this change to mimic that behavior. If we want to update input box to make `SkyThemeService` optional, I'm fine with that as well, but there may be other components we may have to chase down besides that one.